charts/nri-metadata-injection/values.yaml

```diff
@@ -1,81 +1,100 @@
-# IMPORTANT: The Kubernetes cluster name
-# https://docs.newrelic.com/docs/kubernetes-monitoring-integration
-# It can also be set as global so that it is shared by other newrelic product's charts.
-# cluster: ""
+# -- Override the name of the chart
+nameOverride: ""
+# -- Override the full name of the release
+fullnameOverride: ""
 
-# Enable the metadata decoration only for pods living in namespaces labeled
-# with 'newrelic-metadata-injection=enabled'.
-injectOnlyLabeledNamespaces: false
+# -- Name of the Kubernetes cluster monitored. Can be configured also with `global.cluster`
+cluster: ""
+# -- This set this license key to use. Can be configured also with `global.licenseKey`
+licenseKey: ""
```

Comment:
license key is not used in this chart, also customSecretName and customSecretLicenseKey should not be set

Reply:
Unless something changed

Reply:
You are right. Solved in [#4a082e6](https://github.com/newrelic/k8s-metadata-injection/pull/171/commits/4a082e6389a9968fb77a01508b68f2da71551ca0)